### ISSUE TITLE
Flycut freezes on screen

### ISSUE THREAD

**Issue by zum1** [TermiT/Flycut#198]
Fairly often Flycut freezes on my screen i.e. sticky bezel is so sticky it won't go away unless I quit Flycut.

**Comment by douglsmith** [comment]
I can make this happen repeatedly when I invoke Flycut with Command-Option-V, then change my mind and click outside of the overlay. I have found two ways to make the overlay go away: 1. Quit Flycut; or 2. Choose `Clear All` from the menu and then click outside of the overlay.

I'm not certain, but I think this behavior began with macOS Catalina.

**Comment by MarkJerde** [comment]
Hi @douglsmith.
From your description, I think this is just how FlyCut has worked for as long as I am aware. Try clicking back inside Flycut’s bezel and then you should be able to use the Esc key to close it. It’s one of those quirks (bugs) that would be nice to fix, but hasn’t been disruptive enough to me for me to spend the time. 🙂 

Hope this helps,
Mark

**Comment by zum1** [comment]
For me to keeps me from using sticky bezel as it won’t go away unless i quit the app.

**Comment by douglsmith** [comment]
Yes, I can click back inside the bezel and the Esc out, at least in a quick test. Good to know.

**Comment by MarkJerde** [comment]
I was going to say this would be a good task for anybody wanting to learn some macOS development (and deal with really really old manual memory management), but then just decided it would be fun to fix it myself. 😄
